packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -12,11 +12,17 @@ import { orbitMainnets, orbitTestnets } from './orbitChainsList'
 
 export const getChains = () => {
   const chains = Object.values(arbitrumSdkChains)
-  return chains.filter(
-    chain =>
-      // exclude devnet
-      chain.chainID !== 1338
-  )
+  return chains.filter(chain => {
+    // exclude devnet
+    if (chain.chainID === 1338) {
+      return false
+    }
+    // exclude L1 chains with no child chains
```

Comment:
This is the case for Holesky. It doesn't have any default child chains, but users can add a custom Orbit chain with Holesky as the parent chain.

Only then we want to show it.